packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -130,7 +132,7 @@ const PatientInfo: React.FC<PatientInfoProps> = ({ patient }) => {
 
 const VisitHeader: React.FC = () => {
   const { t } = useTranslation();
-  const { patient } = usePatient();
+  const { patient } = usePatientOrOfflineRegisteredPatient(getPatientUuidFromUrl());
```

Comment:
This leads me to think about @ibacher's comment [here](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1083#discussion_r1153417779). I think he set the stage for migrating away from this hook and integrating its logic into `usePatient` at some point. Dunno if migrating to `usePatient` here breaks things at the moment, but I'm inclined towards doing so.